opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -164,11 +186,23 @@ def _transcribe(item, parent=None, editRate=24, masterMobs=None):
     elif isinstance(item, aaf.component.SourceClip):
         result = otio.schema.Clip()
 
-        # ref = item.resolve_ref()
-        # name = ref.name
+        mobs = [item.resolve_ref()]
+        start = item.start_time
+
+        for c in item.walk():
+            mob = c.resolve_ref()
+            start += c.start_time
+            if mob:
+                mobs.append(mob)
+
+        # Evidently the last mob is the one with timecode
+        timecode = _extract_timecode(mobs[-1])
```

Comment:
I think we should check the mobs list size before accessing it via [-1], I believe an empty list will create an index error in that case.

Reply:
mobs is initialized to `[item.resolve_ref()]` which I would imagine would always be valid?  But I can add a check, good note.